pythoncode/antDongle.py

```diff
@@ -242,7 +243,8 @@
                                         #           0x04 = Global datapages used
 TransmitPower_0dBm      = 0x03          # 9.4.3     Output Power Level Settings
 RfFrequency_2457Mhz     =   57          # 9.5.2.6   Channel RF Frequency
-RfFrequency_2466Mhz     =   66          # used for Tacx i-Vortex only
+RfFrequency_2460Mhz     =   60          # used for Tacx i-Vortex only
+RfFrequency_2466Mhz     =   66          # used for Tacx i-Genius only

```

Comment:
are the code comments here the wrong way round? ie 2460-genius and 2466-vortex?

Reply:
Yes, I think you're right. The comments would suggest that the frequency for the i-Vortex has changed which is not correct. I will change this. Thank you!